libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewConfig.m

```diff
@@ -133,21 +140,31 @@ - (BOOL)validate:(NSError **)error {
         return NO;
     }
     
-    // Sanity check local URLs against absolute URL values.
-    if (self.isLocal && self.startPageIsAbsoluteUrl) {
-        [[self class] createError:error withCode:SFSDKHybridAppConfigErrorCodeLocalPageAbsoluteURL message:[SFSDKResourceUtils localizedString:@"appConfigValidationErrorLocalAbsoluteURL"]];
+    // startPage must be a relative URL.
+    if ([[self class] urlStringIsAbsolute:self.startPage]) {
+        [[self class] createError:error withCode:SFSDKHybridAppConfigErrorCodeStartPageAbsoluteURL message:[SFSDKResourceUtils localizedString:@"appConfigValidationErrorStartPageAbsoluteURL"]];
```

Comment:
I'm curious, why are you calling static methods with [MyClass meth] instead of [[self class] meth]? Is it to be able to override them in subclasses?

Reply:
Force of habit, mostly.  I can make the explicit calls for these.  I don't expect they'll be overridden in these cases.